### ISSUE TITLE
move next page without a checked checkbox.

### ISSUE THREAD

**Issue by makeajourney** [dasbeerboot/Bday-gift-quiz-web#14]
on the first page,
It can move to the next page when i pressed "start" without a checked checkbox.
I think it should not move to the next page.

**Comment by dasbeerboot** [comment]
You’re right I put it as required but it still redirects to next page so I’m trying to fix it now
